app/services/populations/defaults_from_nomis.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Populations
+  class DefaultsFromNomis
+    def self.call(location, date)
+      nomis_agency_id = location.nomis_agency_id
+      assigned_cells = NomisClient::Rollcount.get(nomis_agency_id, false)
```

Comment:
small point: it might be a bit clearer if the service could be invoked with
` NomisClient::Rollcount.get(nomis_agency_id, :assigned)`
and
` NomisClient::Rollcount.get(nomis_agency_id, :unassigned)`
rather than true/false

Reply:
Hmm... not really sure how that would work in practice. I could add an `assigned_or_unassigned` param to the `get` method and then check the value of that `== :assigned` but it doesn't feel like it would be cleaner that way.

If the boolean param is an issue my preference would be to split this into two methods named `get_assigned` and `get_unassigned` within the `Rollcount` class, but that might cause some knock on issues working with the Nomis Oauth test stubs. I can experiment with that if the boolean param is a big issue?

Reply:
its a small concern so not worth spending much time over. But if `get_assigned` and `get_unassigned` is possible that would be good too 👍 
my worry is it is not obvious what `NomisClient::Rollcount.get(nomis_agency_id, false)` means without looking at the definition of the method.

Reply:
Okay, so if the concern is around argument clarity I'll change them to named arguments instead as the simplest fix. Some people have a dislike of any boolean arguments at all so I thought that was what you meant. Updates soon!

Reply:
yep, named args would be much clearer 👍